chalicelib/app_utils.py

```diff
@@ -602,7 +607,7 @@ def view_foursight_history(environ, check, start=0, limit=25, is_admin=False,
     if server is not None:
         favicon = get_favicon(server)
     else:
-        favicon = DEFAULT_FAVICON
+        favicon = FOURFRONT_FAVICON
```

Comment:
Even if FOURFRONT is the right default, why is the variable name not `DEFAULT_FAVICON` or at least `DEFAULT_DEFAULT_FAVICON` (presumably for when ecosystem-based defaulting has failed). Though this is often a place where having a third icon that is the "I couldn't find one" icon makes bugs get fixed quicker.

Anyway, this function looks suspiciously Foursight-centric in a set of changes that seem to be going the other way.